bindings/pyroot/pythonizations/CMakeLists.txt

```diff
@@ -113,7 +123,23 @@ set(cpp_sources
 set(ROOTPySrcDir python/ROOT)
 set(ROOT_headers_dir inc)
 
-file(COPY ${ROOTPySrcDir} DESTINATION ${localruntimedir})
+# Add custom rules to copy the Python sources into the build directory
```

Comment:
Perhaps @bellenot can have a look at this CMake changes once he is back.

Reply:
Sure!